### DISCUSSION TITLE
broadlink e  HASSIO

### DISCUSSION THREAD

**Discussion by RobyBS** [indomus/forum#1504] [HUB: Home Assistant]
Buongiorno a tutti ... 

come faccio ad importare dei comandi IR o RF ( RM4PRO ) avendo HASSIO su raspberry ed un pc a disposizione con solo windows ? 

scusate se la domanda è stupida , ma vedo solo risposte con installazione docker che presumo non posso utilizzare avendo HASSIO su raspberry dedicato ... 

grazie
Roby

**Comment by indomus** [comment]
https://indomus.it/componenti/broadlink-manager/

**Reply by RobyBS** [reply]
grazie ...
non riesco a collegare il mio RM4pro con la app e-control ... posso procedere con app broadlink ufficiale, sempre fermandomi al momento dell'assegnazione dell'IP sulla mia wifi ? ... e poi procedere con la registrazione dei comandi che mi interessano ..

**Reply by indomus** [reply]
Fai un test, tanto danni non ne puoi fare.

**Reply by RobyBS** [reply]
sembra che non ne vengo a capo ... inserisco ilmio RM PRO4 sulla mia rete Wifi ... ma non riesco a captare il segnale RF nè con broadlink manager , nè con HA con il comando remote.learn_command ...
sono sconfortato ...

esiste un altro modo per acquisire segnali RF ? anche con un altro apparato ... 


ultima info ... per "domotizzare" ( inteso come apprendere per poi riutilizzare a mio piacimento ) segnali sempre fissi e non rolling che viaggiano su frequenza di 800 mhz ( motorizzazione ante con bracci della casa ) come potrei fare  ?

**Reply by indomus** [reply]
https://indomus.it/guide/catturare-codici-infrarossi-e-radiofrequenza-usando-broadlink-manager-via-web-gui/

**Reply by RobyBS** [reply]
..uso hassio su raspberry dedicato ... :( ...

**Reply by indomus** [reply]
non hai un computer windows sul quale appoggiarti?

**Reply by RobyBS** [reply]
non mi sembra di aver capito dal link che posso installare docker su un pc windows ... la guida ti rimanda (per il caso di windows ) all'installazione di broadlink manager che purtroppo non mi funziona con RM4PRO ( e non posseggo RM3mini che dicono vada ancora bene con quel tipo di software ) ... 
capisco male ... ? ...

**Reply by indomus** [reply]
https://indomus.it/guide/catturare-codici-infrarossi-e-radiofrequenza-usando-broadlink-manager/
